src/components/ExplorePage/index.js

```diff
@@ -76,6 +76,9 @@ const useStyles = makeStyles(
         zIndex: zIndex.appBar,
       },
     },
+    mobileTabs: {
+      top: 80,
+    },
```

Comment:
We shouldn't need.

Reply:
this could have been done in the component, but it would make the storybook awkward.
I positioned the tabs fixed @kilemensi

Reply:
> **shouldn't** 
>
As in if I understood your code correct, there is not need to position the tabs. The tab should appear in the right place if something in the code wasn't push it around.

If I had to guess, it is this lazy assignment you're making [here](https://github.com/CodeForAfrica/PesaYetu/pull/114/files#diff-41bdbf52409bf796e3ea8f7fbb2b414165b92eda1d8ff4011c6e62100ddbb76eR35). We never assign `classes` this way because this is bound to have side-effects... Just manually pick the ones you need `classes={{ tabs: classes.tabs, tab: .., tabSelected: ... }}`